programmatic_execution_of_notebooks/3-explore_sample_qc_results.ipynb

```diff
@@ -0,0 +1,1246 @@
+{
```

Comment:
Can you please remove this file since its just a renamed copy of https://github.com/verilylifesciences/variant-qc/blob/main/notebooks/ExploreSampleQCResults.ipynb?